src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -171,6 +171,7 @@ public void keyboardShortcutsTest() {
         assertEquals(repoSelectorComboBox.getValue(), "dummy/dummy");
         // test shortcut when focus is on panel
         PlatformEx.waitOnFxThread();
+        sleep(1000);
```

Comment:
The time waited being nondeterministic is fine, as long as it always eventually finds the node. Is that possible?